packages/react-admin-date-picker/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "@vivid-planet/react-daterange-picker",
+    "version": "1.0.0-alpha.1",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "files": [
+        "lib"
+    ],
+    "dependencies": {
+        "moment": "^2.24.0",
```

Comment:
moment.js is not an option as dependency, use date-fns instead.

Reply:
I agree on that, but I believe the package used for the date pickers requires moment. 🙄

Reply:
@johnnyomair you're right, date-fns is needed for the picker